docs/docs/reference/configuration.md

```diff
@@ -41,7 +41,7 @@ For default values and the structure of the yaml file please check file - `confi
 Usage example:
 
 ```sh
-cp config.yaml.example config.yaml
+cp config/config.yaml config.yaml
```

Comment:
having config.yaml at 2 places can be confusing. I think having an example file expresses intent.